src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/læremidler/beregning/LæremidlerPeriodeUtil.kt

```diff
@@ -11,11 +11,11 @@ object LæremidlerPeriodeUtil {
      * Splitter en periode i løpende måneder. Løpende måned er fra dagens dato og en måned frem i tiden.
      * eks 05.01.2024-29.02.24 blir listOf( P(fom=05.01.2024,tom=04.02.2024), P(fom=05.02.2024,tom=29.02.2024) )
      */
-    fun <P : Periode<LocalDate>, VAL : Periode<LocalDate>> P.splitPerLøpendeMåneder(medNyPeriode: (fom: LocalDate, tom: LocalDate) -> VAL): List<VAL> {
+    fun <P : Periode<LocalDate>, VAL : Periode<LocalDate>> P.splitPerLøpendeMåneder(medNyPeriode: (fom: LocalDate, tom: LocalDate) -> VAL, skalKutteSistePeriode: Boolean = true): List<VAL> {
```

Comment:
Minor detail, men hvis man legger default variabeln først, og lambda sist så kan man fortsatt bruke 
`splitPerLøpendeMåneder {}` med `{}`
```suggestion
    fun <P : Periode<LocalDate>, VAL : Periode<LocalDate>> P.splitPerLøpendeMåneder(
        skalKutteSistePeriode: Boolean = true,
        medNyPeriode: (fom: LocalDate, tom: LocalDate) -> VAL
     ): List<VAL> {
```

Reply:
Kuuul 🤩 Det visst jeg ikke!